src/main/java/frc/robot/RobotContainer.java

```diff
@@ -157,7 +155,7 @@ private void configureButtonBindings() {
     coDriverStick.POV_South
         .whenPressed(() -> sub_shooter.setGoalRPM(ShooterPrefs.shooterPresetUpperLaunchpadRPM.getValue()));
 
-    coDriverStick.POV_West.whileHeld(() -> sub_hood.setAngleDegrees(HoodPrefs.hoodTarmac), sub_hood);
+    coDriverStick.POV_West.whenPressed(() -> sub_hood.setAngleDegrees(HoodPrefs.hoodTarmac), sub_hood);
```

Comment:
this is the only one of the POVs that passes the subsystem as required. which is the correct behavior?